notebooks/Satellite_Data/PlottingSatelliteData.ipynb

```diff
@@ -498,16 +516,39 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# !load solutions/sat_map.py\n",
+    "fig = plt.figure(figsize=(10, 8))\n",
+    "ax = fig.add_subplot(1, 1, 1, projection=proj)\n",
+    "\n",
+    "# Plot the data using imshow\n",
+    "ax.imshow(dat, origin='upper', \n",
+    "          extent=(x.min(), x.max(), y.min(), y.max()))\n",
+    "\n",
+    "# Add country borders and states (use your favorite linestyle!)\n",
+    "ax.add_feature(cfeature.BORDERS, linewidth=2, edgecolor='black')\n",
+    "ax.add_feature(cfeature.STATES, linestyle=':', edgecolor='black')\n",
+    "\n",
+    "# Bonus/Daily Double\n",
+    "timestamp = datetime.strptime(ds.start_date_time, '%Y%j%H%M%S')\n",
     "add_timestamp(ax, timestamp, pretext='GOES-16 Ch.{} '.format(channel),\n",
-    "              high_contrast=True, fontsize=16, y=0.01)\n",
-    "fig"
+    "              high_contrast=True, fontsize=16, y=0.01)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we have two timestamps! Let's remove the old one (this is why we kept a handle to it)."
+    "### Using Colortables\n",
+    "The map is much improved now, but it would look much better with a different color scheme. \n",
+    "\n",
+    "Colormapping in matplotlib (which backs CartoPy) is handled through two pieces:\n",
+    "\n",
+    "- The colormap controls how values are converted from floating point values in the range [0, 1] to colors (think colortable)\n",
```

Comment:
If you flip the order of these bullets, they would read in the order of how the steps are actually applied to convert from their original data to an image. I'm not sure if you had another intention in mind here behind the ordering, though.